src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/Touch/Enums/TouchStatus.shared.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Xamarin.CommunityToolkit.Effects
+{
+	public enum TouchStatus
+	{
+		Started,
+		Completed,
```

Comment:
Would a `Running` value (or similar) make sense to notify when touch started but is still running and not completed or canceled?

Reply:
There are two separate properties (Status and State)

State: Normal or Pressed (I guess, the Pressed state is what you need)
Status: Started, Completed & Canceled (Started means the touch is **running** now)